src/mawpy/io.py

```diff
@@ -0,0 +1,119 @@
+import fsspec
+import os
+import pandas as pd
+import importlib
+from typing import Callable
+
+FUNCTION = "func"
+EXTENSION = "ext"
+
+supported_files = {
+    "csv": {FUNCTION: "pandas.read_csv", EXTENSION: ".csv"},
+    "excel": {FUNCTION: "pandas.read_excel", EXTENSION: ".xlsx"},
+}
```

Comment:
Why not:
import pandas

```suggestion
supported_files = {
    "csv": pandas.read_csv,
    "excel": pandas.read_excel,
}
```

We know we'll use pandas anyway, right?

Reply:
I was thinking of delaying the import, but then really we are using pandas here everywhere... so maybe it's better to go simpler